Libplanet/Store/IBlockStatesStore.cs

```diff
@@ -1,184 +0,0 @@
-        /// <paramref name="sourceChainId"/> does not exist.</exception>
-        /// <seealso cref="IterateStateReferences(Guid, string, long?, long?, int?)"/>
-        /// <seealso
-        /// cref="StoreStateReference(Guid, IImmutableSet{string}, HashDigest{SHA256}, long)"/>
-        void ForkStateReferences<T>(
-            Guid sourceChainId,
-            Guid destinationChainId,
-            Block<T> branchPoint)
-            where T : IAction, new();
-
-        /// <summary>
-        /// Gets the states updated by actions in the inquired block.
-        /// </summary>
-        /// <param name="blockHash"><see cref="Block{T}.Hash"/> to query.
-        /// </param>
-        /// <returns>The states updated by actions in the inquired block.
-        /// If actions definitely do not update any addresses it returns
-        /// an empty map.  If there is no record for the inquired block
-        /// (because actions in it have never been evaluated yet) it returns
-        /// <c>null</c> instead.
-        /// </returns>
-        /// <remarks>It does not return all states built up from the genesis
-        /// block nor delta, but only dirty states by actions the inquired
-        /// block.
-        /// <para>For example, if actions in the genesis block do
-        /// <c>a++; b++</c>, /// and actions in the second block do
-        /// <c>b++; c++</c>, this method /// for the second block returns
-        /// <c>b = 2; c = 1</c> (dirty), not
-        /// <c>a = 1; b = 2; c = 1</c> (all states) nor
-        /// <c>b = 1; c = 1</c> (delta).</para>
-        /// </remarks>
-        /// <seealso cref="SetBlockStates"/>
-        IImmutableDictionary<string, IValue> GetBlockStates(HashDigest<SHA256> blockHash);
-
-        /// <summary>
-        /// Sets the states updated by actions in the specified block
-        /// (i.e., <paramref name="blockHash"/>).
-        /// </summary>
-        /// <param name="blockHash"><see cref="Block{T}.Hash"/> to update states.
-        /// </param>
-        /// <param name="states">The states updated by actions in the specified block
-        /// (i.e., <paramref name="blockHash"/>).
-        /// </param>
-        /// <seealso cref="GetBlockStates"/>
-        void SetBlockStates(
-            HashDigest<SHA256> blockHash,
-            IImmutableDictionary<string, IValue> states
-        );
-
-        /// <summary>
-        /// Prunes states in blockchain <paramref name="chainId"/> with until specified block
-        /// <paramref name="until"/>.
-        /// </summary>
-        /// <param name="chainId">The chain ID to prune block states.</param>
-        /// <param name="until">The upper bound block to prune states.</param>
-        /// <typeparam name="T">An <see cref="IAction"/> class used with
-        /// <paramref name="until"/>.</typeparam>
-        void PruneBlockStates<T>(
-            Guid chainId,
-            Block<T> until)
-            where T : IAction, new();
-    }
-}
```

Comment:
`IBlockStateStore` interface was removed to deprecate *block states, state reference* model.